drivers/sensor/st/Kconfig

```diff
@@ -32,6 +32,7 @@ source "drivers/sensor/st/lsm6dsv16x/Kconfig"
 source "drivers/sensor/st/lsm9ds0_gyro/Kconfig"
 source "drivers/sensor/st/lsm9ds0_mfd/Kconfig"
 source "drivers/sensor/st/qdec_stm32/Kconfig"
+source "drivers/sensor/st/stm32_dts/Kconfig"
```

Comment:
May we use a different acronym to describe digital temperature sensor? e.g: dtemp or similar.

Reply:
Yes, it's possible to rename it into `stm32_dtemp` if people feel it is better like that.

Reply:
I agree we should absolutely avoid dts :)
`stm32_dtemp` could be ok we think there won't be any confusion with die temp (for instance if some other vendor comes up with a vnd_dtemp for his own die temp sensor).
`stm32_digi_temp` is less synthetic but should avoid possible confusion.